From 586515b10cee35fd5aab519a1b8012dfd4751c22 Mon Sep 17 00:00:00 2001
From: Jianxiong Pan <jianxiong.pan@amlogic.com>
Date: Tue, 27 Sep 2022 11:11:52 +0800
Subject: [PATCH] gki: delete vendor modifications to meet gki requirements.
 [1/1]

PD#SWPL-94445

Problem:
vendor modification break gki.

Solution:
delete the vendor modifications.

Verify:
sc2_ah212.

Change-Id: I7f196c0cbd43582e4a887aefe45062210392da81
Signed-off-by: Jianxiong Pan <jianxiong.pan@amlogic.com>
---
 arch/arm64/include/asm/cache.h      | 2 +-
 drivers/dma-buf/heaps/system_heap.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/include/asm/cache.h b/arch/arm64/include/asm/cache.h
index 23bba8790dfc..374ee4c5dc60 100644
--- a/arch/arm64/include/asm/cache.h
+++ b/arch/arm64/include/asm/cache.h
@@ -23,7 +23,7 @@
  * cache before the transfer is done, causing old data to be seen by
  * the CPU.
  */
-#ifdef CONFIG_AMLOGIC_MEMORY_EXTEND
+#ifdef CONFIG_AMLOGIC_MEMORY_OPT
 #define ARCH_DMA_MINALIGN	L1_CACHE_BYTES
 #else
 #define ARCH_DMA_MINALIGN	(64)
diff --git a/drivers/dma-buf/heaps/system_heap.c b/drivers/dma-buf/heaps/system_heap.c
index ae7c5a6df037..b05e50ceed00 100644
--- a/drivers/dma-buf/heaps/system_heap.c
+++ b/drivers/dma-buf/heaps/system_heap.c
@@ -52,7 +52,7 @@ struct dma_heap_attachment {
 #define HIGH_ORDER_GFP  (((GFP_HIGHUSER | __GFP_ZERO | __GFP_NOWARN \
 				| __GFP_NORETRY) & ~__GFP_RECLAIM) \
 				| __GFP_COMP)
-#ifdef CONFIG_AMLOGIC_MEMORY_EXTEND
+#ifdef CONFIG_AMLOGIC_MEMORY_OPT
 #define LOW_ORDER_GFP_NO_WARN (GFP_HIGHUSER | __GFP_ZERO | __GFP_COMP | __GFP_NOWARN)
 static gfp_t order_flags[] = {HIGH_ORDER_GFP,
 				LOW_ORDER_GFP_NO_WARN, LOW_ORDER_GFP_NO_WARN,
-- 
2.25.1

